helm/default-apps-cloud-director/values.schema.json

```diff
@@ -1,125 +1,116 @@
-                    },
-                    "namespace": {
-                        "minLength": 1,
-                        "type": "string"
-                    },
-                    "version": {
-                        "minLength": 1,
-                        "type": "string"
                     }
+                }
+            }
+        }
+    },
+    "type": "object",
+    "properties": {
+        "apps": {
+            "type": "object",
+            "properties": {
+                "certExporter": {
+                    "$ref": "#/$defs/app"
+                },
+                "clusterResources": {
+                    "$ref": "#/$defs/app"
                 },
-                "required": [
-                    "appName",
-                    "catalog",
-                    "chartName",
-                    "clusterValues",
-                    "forceUpgrade",
-                    "namespace",
-                    "version"
-                ]
-            },
-            "type": "object"
+                "metricsServer": {
+                    "$ref": "#/$defs/app"
+                },
+                "netExporter": {
+                    "$ref": "#/$defs/app"
+                },
+                "nodeExporter": {
+                    "$ref": "#/$defs/app"
+                },
+                "observabilityBundle": {
+                    "$ref": "#/$defs/app"
+                }
+            }
         },
         "clusterName": {
-            "minLength": 1,
             "type": "string"
         },
         "managementCluster": {
-            "minLength": 1,
             "type": "string"
         },
         "organization": {
-            "minLength": 1,
             "type": "string"
         },
         "userConfig": {
             "type": "object",
             "properties": {
-                "nodeExporter": {
-                    "type": "object",
-                    "properties": {
-                        "configMap": {
-                            "type": "object",
-                            "properties": {
-                                "values": {
-                                    "type": "string"
-                                }
-                            }
-                        }
-                    }
+                "netExporter": {
+                    "$ref": "#/$defs/userConfig"
                 },
-                "certManager": {
-                    "type": "object",
-                    "properties": {
-                        "configMap": {
-                            "type": "object",
-                            "properties": {
-                                "values": {
-                                    "type": "string"
-                                }
-                            }
-                        }
-                    }
+                "nodeExporter": {
+                    "$ref": "#/$defs/userConfig"
                 }
             }
         }
-    },
-    "required": [
```

Comment:
Why do we delete `required` section?

Reply:
It was a copy/paste. I can bring it back